### PR TITLE
ML UserDomain/UnitTest

### PR DESCRIPTION
**Pull request by EverLoSa**
-ML UserDomain
-Netflow-Storyboard: Globe View refactor
-ML unit test cleanup

NOTE: This branch contains multiples commits because we are replicating functionality from the old repo (Open Network Insight)

### CODE REVIEW

**Comment by daortizh** [comment]
+1

**Comment by rabarona** [comment]
Looks good to me but seems like there is still one conflict with.
spot-oa/ui/flow/js/constants/NetflowConstants.js

**Comment by EverLoSa** [comment]
@rabarona , yes, it needs to be merged manually, I already reviewed with Diego he did the commit. If the PR is approved I will proceed with the manually merge.

**Comment by rabarona** [comment]
Perfect.

**Comment by brandon-edwards** [comment]
+1 Looks good to me

File: spot-ml/src/test/scala/org/apache/spot/netflow/FlowWordCreatorTest.scala [hunk 1]
```diff
@@ -39,7 +37,7 @@ class FlowWordCreatorTest extends FlatSpec with Matchers {
 
 
     dstWord shouldBe "-1_23_5_2_0"
-    srcWord shouldBe "23_5_2_0"
+    srcWord shouldBe  "23_5_2_0"
```
Comment (natedogs911) [inline spot-ml/src/test/scala/org/apache/spot/netflow/FlowWordCreatorTest.scala]: can we drop the rogue space here?

**Comment by natedogs911** [comment]
+1

File: spot-ml/INSTALL.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Names and language that we will use from the configuration variables for Spot (t
 - MLNODE The node from which the spot-ml routines are invoked
```
Comment (NathanSegerlind) [inline spot-ml/INSTALL.md]: maybe this is a ticket for future work... but we have the opportunity to greatly simplify ML installation and execution (and rid ourselves of the concept of an ML node)

**Comment by NathanSegerlind** [comment]
FYI,  can't run the unit tests in the master branch without this checkin

**Pull request merged**